src/index.html

```diff
@@ -11,8 +11,51 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__photo"></div>
+      <p class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="card__code">Product code: 195434</p>
+      <div class="stars">
+        <div class="stars__total stars__total--4">
```

Comment:
The 'stars' block should be reusable and independent. Therefore, you should not use 'stars__total--4' class to indicate the number of active stars. The number of active stars should be set by adding 'stars--n' class to the 'stars' block where 'n' is the number of active stars. For example, 'stars stars--4'.